presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -221,10 +221,20 @@ MySQL Connector Limitations
 The following SQL statements are not yet supported:
 
 * :doc:`/sql/delete`
-* :doc:`/sql/alter-table`
-* :doc:`/sql/create-table` (:doc:`/sql/create-table-as` is supported)
+* :doc:`/sql/create-table` with PRIMARY KEY
 * :doc:`/sql/grant`
 * :doc:`/sql/revoke`
 * :doc:`/sql/show-grants`
 * :doc:`/sql/show-roles`
 * :doc:`/sql/show-role-grants`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/update`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/analyze`
+
+The following supported SQL statements with certain conditions:
+
+* :doc:`/sql/alter-table`: Supports adding a new column to an existing table.
+* :doc:`/sql/create-table` without PRIMARY KEY: Supports creating tables without defining a primary key.
+* :doc:`/sql/create-table-as`: Supports creating a new table using the results of a select statement.
+* :doc:`/sql/truncate-table`: Supports truncation of a table.
```

Comment:
```suggestion
* :doc:`/sql/truncate`: Supports truncation of a table.
```
Fixed the incorrect link. When you can, test your doc draft by doing a local build of the docs which can help find problems like this.